### PR TITLE
feat: Creating a zip file rather coping all individual log files

### PR DESCRIPTION
**Pull request by rjaini**
Creating a zip file rather coping all individual log files

### CODE REVIEW

File: registry/diagnosis/collectlogs.sh [hunk 1]
```diff
@@ -1,28 +1,49 @@
...
+    echo "[$(date +%Y%m%d%H%M%S)][INFO] Compressing logs and cleaning up temp files"
+    CURRETUSER=$(whoami)
+    LOGFILENAME="${FILENAME}.zip"
+    sudo rm -f ${LOGFILENAME}
+    
+    sudo chown -R ${CURRETUSER} ${LOGDIRECTORY}
+    if [ -f /opt/azure/vhd-install.complete ]; then
+        echo "[$(date +%Y%m%d%H%M%S)][INFO] aks base image; skipping dependencies installation"
+    else
+        echo "[$(date +%Y%m%d%H%M%S)][INFO] installing zip module"
```
Comment (jadarsie) [inline registry/diagnosis/collectlogs.sh]: zip package, not module

Comment (jadarsie) [inline registry/diagnosis/collectlogs.sh]: personal preference, I like sentence case (capitalize first letter)

File: registry/diagnosis/collectlogs.sh [hunk 2]
```diff
@@ -1,28 +1,49 @@
...
-    sudo cp "$f" ${LOGDIRECTORY}/reg-"${f%}" || :
+    sudo cp "$f" ${LOGDIRECTORY}/var/log/azure/reg-"${f%}" || :
 done
 
-cd ${LOGDIRECTORY}
+#sudo cp /var/log/cloud-init.log ${LOGDIRECTORY}/reg-cloud-init.log || :
+#sudo cp /var/log/waagent.log ${LOGDIRECTORY}/reg-waagent.log || :
+
+mkdir -p ${LOGDIRECTORY}/containers
+echo "[$(date +%Y%m%d%H%M%S)][INFO] Collect registry container logs."
```
Comment (jadarsie) [inline registry/diagnosis/collectlogs.sh]: `CollectING`? just for consistency

File: registry/diagnosis/collectlogs.sh [hunk 3]
```diff
@@ -1,28 +1,49 @@
...
-if [ -d "$LOGDIRECTORY" ]; then
-    NOW=`date +%Y%m%d%H%M%S`
-    mv $LOGDIRECTORY ${LOGDIRECTORY}-${NOW}
-fi
+compressLogsDirectory()
+{
+    sync
+    
+    echo "[$(date +%Y%m%d%H%M%S)][INFO] Compressing logs and cleaning up temp files"
+    CURRETUSER=$(whoami)
```
Comment (Bhuvaneswari-Santharam) [inline registry/diagnosis/collectlogs.sh]: misspelled:CURRENTUSER

File: registry/diagnosis/collectlogs.sh (hunk 1 above)
Comment (rjaini) [reply registry/diagnosis/collectlogs.sh]: updated.

Comment (rjaini) [reply registry/diagnosis/collectlogs.sh]: updated.

File: registry/diagnosis/collectlogs.sh (hunk 2 above)
Comment (rjaini) [reply registry/diagnosis/collectlogs.sh]: updated.

File: registry/diagnosis/collectlogs.sh (hunk 3 above)
Comment (rjaini) [reply registry/diagnosis/collectlogs.sh]: Fixed. Also in k8s log collection.

**Pull request merged**